docs/apache-airflow/issue-triage-process.rst

```diff
@@ -0,0 +1,195 @@
+
+We recently introduced ``kind:task`` to categorize issues which are
+identified elements of work to be done, primarily as part of a larger
+change to be done as part of an AIP or something which needs to be cleaned
+up in the project.
+
+Issues of ``kind:documentation`` are for changes which need to be
+made to the documentation within the project.
+
+
+**Area**
+
+The “area” set of labels should indicate the component of the code
+referenced by the issue. At a high level, the biggest areas of the project
+are: Airflow Core and Airflow Providers, which are referenced by ``area:core``
+and ``area:providers``. This is especially important since these are now
+being released and versioned independently.
+
+There are more detailed areas of the project such as Scheduler, Webserver,
+API, UI, Logging, and Kubernetes, which are all conceptually under the
+“Airflow Core” area of the project.
+
+Similarly, some providers are broken out such as Apache (``provider/Apache``),
+AWS (``provider/AWS``), Azure (``provider/Azure``), and Google (``provider/Google``).
+These make it easier for developers working on a single provider to
+track issues for that provider.
+
+Most issues need a combination of "kind" and "area" labels to be actionable.
+For example:
+
+* Feature request for an additional API would have ``kind:feature`` and ``area:API``
+* Bug report on the User Interface would have ``kind:bug`` and ``area:UI``
+* Documentation request on the Kubernetes Executor, would have ``kind:documentation`` and ``area:kubernetes``
+
+
+Response to issues
+''''''''''''''''''
+
+Once an issue has been created on the Airflow project, someone from the
+Airflow team or the Airflow community typically responds to this issue.
+This response can have multiple elements.
+
+**Priority**
+
+After significant discussion about the different priority schemes currently
+being used across various projects, we decided to use a priority scheme based
+on the Kubernetes project, since the team felt it was easier for people to
+understand.
+
+Therefore, the priority labels used are:
+
+* ``priority:critical``: Showstopper bug that should be resolved immediately and a patch issued as soon as possible. Typically, this is because it affects most users and would take down production systems.
+* ``priority:high``: A high priority bug that affects many users and should be resolved quickly, but can wait for the next scheduled patch release.
+* ``priority:medium``: A bug that should be fixed before the next release, but would not block a release if found during the release process.
+* ``priority:low``: A bug with a simple workaround or a nuisance that does not stop mainstream functionality.
+
+
+It's important to use priority labels effectively so we can triage incoming issues
+appropriately and make sure that when we release a new version of Airflow,
+we can ship a release confident that there are no “production blocker” issues in it.
```

Comment:
We need to clarify if the intention to use priority labels only on Airflow core or also on providers.
in general issue can be critical in a specific provider (broken provider, broken specific package in provider etc..)
If we set `priority:critical` on Google provider. Does this means we expect/suggest/consider a patch release for this provider?

Reply:
I believe yeah, this might trigger us to have an out-of-band release in case we find a critical issue - especially that we can do it easily now and nothing stops us except the efficiency/batching of voting process.

Reply:
Thank you, great point. done.